pkg/test/e2e/api/cluster_test.go

```diff
@@ -41,7 +41,7 @@ func getKubernetesVersion() string {
 	if len(version) > 0 {
 		return version
 	}
-	return "v1.14.2"
+	return "v1.18.8"
```

Comment:
I accidentally found this ancient and now even unsupported default when grepping through the repo. For all intents and purposes we're already defaulting to 1.18.8 for the e2e tests already.